### ISSUE TITLE
Tekken 3 plays fine but doesn't render correctly

### ISSUE THREAD

**Issue by rev666** [DKWDRV/DKWDRV#13]
Menus and FMV's are fine, but characters in game are missing graphics and flickering. Other than that, no issues. It's playable.
Known issue: XA/CDDA audio doesn't play.

![rsz_tekken3-1](https://github.com/DKWDRV/DKWDRV/assets/6900547/44bc9eb5-21df-489a-9306-6a36a79ff727)
![rsz_tekken3-2](https://github.com/DKWDRV/DKWDRV/assets/6900547/a26ace03-a6c1-4393-a5b1-4c5a75f621c8)
![rsz_tekken3-3](https://github.com/DKWDRV/DKWDRV/assets/6900547/f7a2ff26-f44e-41b8-8d79-99952eb3f2d1)
![rsz_tekken3-4](https://github.com/DKWDRV/DKWDRV/assets/6900547/6d02fe57-3134-4886-b930-f0ce814c4939)
![rsz_tekken3-5](https://github.com/DKWDRV/DKWDRV/assets/6900547/68316bf5-9c18-4c13-b541-4f3cea186024)

Default options on DKWDRV v1.7.5
ROM defragmented
PS2 model SCPH-77006

**Comment by ButtonBasher92** [comment]
I can confirm this, I have the same graphical issues with Tekken 3 and the pause menu doesn't show in game for me. SCPH-90004.

**Comment by Ghaleon7771** [comment]
Something similar happens with Chrono Cross with the character and menu models in the status menu. Its rendering in 480i in those scenarios as well, maybe its just coincidence? all other 480i games i tested were working fine.

**Comment by DKWDRV** [comment]
Does this game flicker from CD with latest version?

**Comment by TetrisTyp** [comment]
> Does this game flicker from CD with latest version?

Yes it does. Since my original CD is broken, i burned the Game and had to use the swap-disc method for the first time ever lol
Starts fine, but same problem as with the USB loaded game. So it is no USB related issue.

Something to add: If you change the Y-Position of the Screen with your build in button-combinations by just one pixel, the visible problem changes. So it seems it has something to do with the individual lines.

**Comment by DKWDRV** [comment]
> > Does this game flicker from CD with latest version?
> 
> Yes it does. Since my original CD is broken, i burned the Game and had to use the swap-disc method for the first time ever lol Starts fine, but same problem as with the USB loaded game. So it is no USB related issue.
> 
> Something to add: If you change the Y-Position of the Screen with your build in button-combinations by just one pixel, the visible problem changes. So it seems it has something to do with the individual lines.

Mechapwn? 
Are you using HDMI? What happens in composite?

**Comment by TetrisTyp** [comment]
No Mechapwn, just normal swap trick.
Right now i use Component Output -> GBS-Control -> HDMI
I dont think it makes a difference, but i can try to find my old composite cable. Will report back - I hope i still have it.

Using the stock psx driver, everything looks fine. One difference - DKWDRV seems to have higher output resolution, right? Because on stock, i am able to activate scanlines in gbs-control (only works <= 240p).
Maybe there is a Problem in upscaling in DKWDRV? Is it possible to lower the output resolution for a test?

Im glad i can help in any way :)

**Comment by TetrisTyp** [comment]
> What happens in composite?

Sorry for the delay.
Tested Composite directly PS2 -> TV without any scaler in between - still the same Problem. CD or USB does'nt matter, same result.

I uploaded 2 videos.

First one shows, how the game runs on original driver. Shows all HUD elements correctly, no missing lines, everything ok.
Second is DKWDRV. Missing HUD elements, flickering, menu changing between solid and lines. And at the end i used the up and down button combination to move the picture up and down, resulting in completely disappearing 3D geometry.

I hope this is helpful to find the issue.

https://github.com/user-attachments/assets/1f844351-4c92-47a1-9215-b234995e19a1

https://github.com/user-attachments/assets/32097d95-57f3-443f-9d9e-5b68e5857e99

**Comment by DKWDRV** [comment]
@TetrisTyp greatest help would be a report of this game on PGIF consoles.

Another thing, on bootup at this [time ](https://youtu.be/DLyQdwQce2k?t=34) do you see a blue dot on top left corner?

**Comment by TetrisTyp** [comment]
> [@TetrisTyp](https://github.com/TetrisTyp) greatest help would be a report of this game on PGIF consoles.

I can't test PGIF, my phat ps2 unfortunately is 8hours away... Maybe someone else can test it...

> Another thing, on bootup at this [time ](https://youtu.be/DLyQdwQce2k?t=34) do you see a blue dot on top left corner?
Yes a blue dot is there, CD + USB on DKWDRV.

![Image](https://github.com/user-attachments/assets/1ef2da36-2fe1-4672-9c47-bbaac433775e)

Anything else i can help with?

**Comment by DKWDRV** [comment]
Thank you @TetrisTyp!
You can try games with with [graphics](https://github.com/DKWDRV/DKWDRV/labels/graphics) and [slow ](https://github.com/DKWDRV/DKWDRV/labels/usb_code_slow_main_lopp) from CD and see if they are broken same way.

**Comment by TetrisTyp** [comment]
No problem - hope you don't get frustrated @DKWDRV.
I will try https://github.com/DKWDRV/DKWDRV/issues/95 Looney Tunes Sheep Rider USA SLUS-01369 next from CD. Will report back later.

**Comment by DKWDRV** [comment]
Is that blue dot for original default CD driver too?

**Comment by TetrisTyp** [comment]
> Is that blue dot for original default CD driver too?

yes

**Comment by DKWDRV** [comment]
Just wanted to confirm that the dot shows on both original driver and DKWDRV. You can ignore it from now.
What I need now is a report from a PGIF console.

**Comment by TetrisTyp** [comment]
Allright.
It will take some days until i get my phat ps2. Hope its not weeks... will report back.

**Comment by TetrisTyp** [comment]
Tested Tekken 3 again via USB (this time exFAT) with the latest version - Looks very very good, no missing triangles, no missing overlay text or menu.

The only thing i noticed (maybe that helps you) is, that when i move the image with the buttoncombination up or down by a pixel, the geomitry is missing again. So - everything is fine, as long as you dont change the position of the picture.

Again: Awesome Update!

**Comment by DKWDRV** [comment]
Latest version should have this fixed for good!
